backend/long_running_request_actors.py

```diff
@@ -84,6 +84,16 @@ def recruit_project_invite_github_collaborators_to_repo(project_id):
     project.invite_github_collaborators_to_repo()
 
 
+@actor(max_retries=3)
+def invite_collaborators_for_team_projects(team_name):
+    from curriculum_tracking.models import RecruitProject
+
+    projects = RecruitProject.objects.filter(recruit_users__groups__name=team_name)
+
+    for project in projects:
+        recruit_project_invite_github_collaborators_to_repo.send(project.pk)
```

Comment:
This can get VERY heavy. There might be a lot of repos here. Is there a way to make the inclusion of repos on complete projects optional? The default behavior should be to not include repos on projects that have completed.